src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build your Astro site with any code editor, installed locally or in the cloud!
+
+[VS Code](https://code.visualstudio.com) is a popular code editor that can be dowloaded locally, and is also used by online platforms such as [GitHub.dev](https://github.dev), [GitHub Codespaces](https://github.com/features/codespaces) and [Gitpod](https://gitpod.io).
+
+
+## VS Code Extension
+
+Astro has its own [Astro VS Code Extension on Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) that you can install in any of these VS Code environments for .`astro` syntax highligting, code completion, and more!
```

Comment:
In web environments, currently the only supported feature is the syntax highlighting :sweat_smile:

Everything else is locked behind https://github.com/withastro/language-tools/issues/180

Reply:
Yes, I think that's all I get in CodeSandbox, so I've specifically said only syntax highlighting below in the online editors setion.  Gitpod gives me the full extension. (After almost a year coding in monochrome, without even syntax highlighting, when it started autocompleting stuff on me, it was quite disorienting!)

Reply:
Yup, Gitpod gives me the full version, but github.dev only gives me the syntax highlighting. (I'm assuming that's "VS Code Browser" instead of full "VS Code.")  So, I'll reorganize/update to make things accurate.